spec/services/notification_service_spec.rb

```diff
@@ -187,29 +187,54 @@ def should_not_email(user_id, n)
   end
 
   describe 'Issues' do
-    let(:issue) { create :issue, assignee: create(:user), description: 'cc @participant' }
+    let(:issue) { create :issue, assignee: create(:user),
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.